lib/rake/build.rake

```diff
@@ -10,13 +10,6 @@ namespace :build do
       HipChat.log 'Applying <b>chef</b> profile...'
       RakeUtils.sudo 'chef-client'
     end
-
-    unless CDO.chef_managed
-      Dir.chdir(aws_dir) do
-        HipChat.log 'Installing <b>aws</b> bundle...'
-        RakeUtils.bundle_install
-      end
-    end
```

Comment:
I think we haven't needed this for years, since we merged our `aws/` Gemfile into the root.